### PR TITLE
phone demo with touch/mouse support

### PR DESCRIPTION
**Pull request by samthor**
@alancutter 

This will probably merge badly with the other demo (the `demos.js` change).

### CODE REVIEW

File: phone/index.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      callbackWhenDone();
+    }
+  };
+
+  // Animate each ball on the main screen moving out in a random direction.
+  timing.easing = 'ease-in';
+  var players = [].slice.call(main.querySelectorAll('.ball')).map(function(ball) {
+    var x = randomInt(-100, +100);
+    var y = randomInt(-400, -200);
+    var deg = randomInt(-25, +25);
```
Comment (alancutter) [inline phone/index.html]: As discussed using random is a bit of a contrived way to show dynamic animation creation. Making the direction based on the relative X coordinate of the mouse/touch event might be a little clearer.

Comment (samthor) [inline phone/index.html]: PTAL, I've done this now.

**Comment by alancutter** [comment]
Nice, I like this a lot better!

I wish the buttons didn't intersect with the bottom bar.
Perhaps instead of a hint toast below the bar in demo itself said "drag me".
In Chrome clicking the bar when its at the top does nothing, clicking it again plays the dragging down animation but the other items don't animate in, they pop in at the end. In Firefox (native/polyfill) it works fine. Looks like a Chrome bug to resolve first.

**Comment by samthor** [comment]
As discussed in-person with @alancutter it sounds like the Chrome bug is >m52.

**Comment by alancutter** [comment]
Chrome bug fixed in M60.
https://bugs.chromium.org/p/chromium/issues/detail?id=648114

**Comment by alancutter** [comment]
LGTM. Chrome 60 is stable now. Merge when ready. (:

**Pull request merged**